appveyor.yml

```diff
@@ -0,0 +1,63 @@
+# https://ci.appveyor.com/project/ContinuumAnalyticsFOSS/constructor
+
+branches:
+  only:
+    - master
+
+environment:
+  global:
+    PYTHON: "C:\\conda"
+    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+    PYTHON_ARCH: "64" # needs to be set for CMD_IN_ENV to succeed. If a mix
+                      # of 32 bit and 64 bit builds are needed, move this
+                      # to the matrix section.
+
+  matrix:
+    - PYTHON_VERSION: "3.6"
+      CONDA_CANARY: "false"
+    - PYTHON_VERSION: "3.5"
+      CONDA_CANARY: "false"
+    - PYTHON_VERSION: "2.7"
+      CONDA_CANARY: "false"
+    - PYTHON_VERSION: "3.6"
+      CONDA_CANARY: "true"
+    - PYTHON_VERSION: "3.5"
+      CONDA_CANARY: "true"
+    - PYTHON_VERSION: "2.7"
+      CONDA_CANARY: "true"
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - PYTHON_VERSION: "3.6"
+      CONDA_CANARY: "true"
+    - PYTHON_VERSION: "3.5"
+      CONDA_CANARY: "true"
+    - PYTHON_VERSION: "2.7"
+      CONDA_CANARY: "true"
+
+platform:
+  -x64
+
+cache:
+  - "C:\\condacache\\pkgs"
+
+install:
+  # Astropy ci-helpers. See https://github.com/astropy/ci-helpers
+  - "git clone git://github.com/astropy/ci-helpers.git"
+  - "powershell ci-helpers/appveyor/install-miniconda.ps1"
+  - "SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "activate test"
+  # Install dependencies. This include test and lint dependencies
+  - "conda update -n root conda -q --no-pin"
+  - "conda config --add pkgs_dirs C:\\condacache\\pkgs"
+  - if %CONDA_CANARY%==true conda update -n root conda -c conda-canary -q --no-pin
+  - "conda install -n test nsis pillow>=3.1 pytest ruaml_yaml"
```

Comment:
With `constructor 2.0.0` we'd need to use `root` instead of `test` due to the imports from `conda`, right?

Reply:
@mbargull right again :-)

Reply:
Took a quick peak at `astropy/ci-helpers/appveyor/install-miniconda.ps1`: it prepares (i.e., installs a bunch of stuff like `pytest` into) the environment `test`, although we'd want `root` ;)